test/ci/buildkite/test_runner.py

```diff
@@ -125,6 +130,14 @@ def main():
                     iterations = 100
             run_resnet50_from_artifacts('./', arguments.artifacts_dir,
                                         batch_size, iterations)
+    elif (arguments.test_resnet50_infer):
+        if get_os_type() == 'Darwin':
+            sys.exit(1)
```

Comment:
You probably don't need this but, at the very least, print a warning ("RN50 inference test not supported on Darwin") before exiting.

Reply:
Ok